testing/describes.js

```diff
@@ -551,6 +552,10 @@ class RealWinFixture {
           installRuntimeStylesPromise(win);
         }
 
+        if (spec.ampAdCss) {
```

Comment:
added to the typedef. Not sure which docs you want me to add to. Added an inline doc here that says what this does.